src/app/app.component.ts

```diff
@@ -2,13 +2,15 @@ import { Component } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { RouterOutlet } from '@angular/router';
 
+import { HeaderComponent } from './core/components/header/header.component';
```

Comment:
Try to implement _paths_ for importing anything from the core, shared, etc. directories. The import will look like _**@core**/xxx/yyy/qqq.component_

https://stackoverflow.com/questions/43281741/how-can-i-use-paths-in-tsconfig-json

Reply:
I will fix it in the next PR